### DISCUSSION TITLE
Explicitly spell out threat models on the "Security Best Practices" page

### DISCUSSION THREAD

**Discussion by makkes** [fluxcd/flux2#4825] [Proposals]
The [Security Best Practices](https://fluxcd.io/flux/security/best-practices/) page currently explains how to securely configure Flux and under "Rationale" provides a bit of explanation on the background of each setting. However, there is no clear threat model mentioned on that page that would help users understand the exact circumstances under which the misconfiguration of that specific setting could be exploited.

I suggest a more concise threat description (e.g. using the STRIDE technique) to be added to each setting mentioned on the best practices page.

**Comment by makkes** [comment]
@pjbgf wdyt?

**Reply by pjbgf** [reply]
Hey @makkes I think that would make sense.

At the time, we created a [threat model](https://hackmd.io/MRg3KTBtQ-y5a2bw9NsysQ) which is linked within the [Multi-tenancy security self-assessment](https://docs.google.com/document/d/1SluYVDuq-egSTurcnrVRMJw6ecSV65Qtgi10T4WHyYs/edit?pli=1) we created for the CNCF Tag Security. That was primarily focused on multi-tenancy threats and concerns at the time, some of which have since been mitigated.

I definitely think there is value in reviewing it and making it public in a more consumeable format. Although I am really struggling with time lately, ping me and I would be more than happy to review it.
